### DISCUSSION TITLE
Are there any CSS styles I can look at?

### DISCUSSION THREAD

**Discussion by eamonburns** [microsoft/terminal#16427] [Q&A]
I am trying to make a program mimic the styles of the windows terminal settings tab. Is there a CSS file that I can look at? Does Windows terminal even use CSS?

**Comment by zadjii-msft** [comment]
Nope! The Terminal uses WinUI and XAML.

**Reply by eamonburns** [reply]
Ok, good to know. Are there at least places where I can see the color palette?
